app/models/organization.rb

```diff
@@ -21,6 +21,7 @@
 #  zipcode                  :string
 #  created_at               :datetime         not null
 #  updated_at               :datetime         not null
+#  account_request_id       :integer
```

Comment:
Per my earlier comment, since you added an FK here, an initialize method wouldn't even need an argument since it has access to the record already. Something like: `Organization.new(account_request_id).initialize_from_account_request` or something like that?